disnake/guild.py

```diff
@@ -1746,6 +1779,149 @@ async def active_threads(self) -> List[Thread]:
 
         return threads
 
+    async def fetch_scheduled_events(
+        self, *, with_user_count: bool = None
+    ) -> List[GuildScheduledEvent]:
+        """|coro|
+
+        Returns a list of :class:`GuildScheduledEvent` of this guild.
+
+        .. versionadded:: 2.3
+
+        Parameters
+        ----------
+        with_user_count: :class:`bool`
+            Whether to include number of users subscribed to each event.
+
+        Raises
+        ------
+        HTTPException
+            The request failed.
+
+        Returns
+        --------
+        List[:class:`GuildScheduledEvent`]
+            The scheduled events.
+        """
+        raw_events = await self._state.http.get_guild_scheduled_events(
+            self.id, with_user_count=with_user_count
+        )
+        return [GuildScheduledEvent(state=self._state, data=data) for data in raw_events]
+
+    async def fetch_scheduled_event(
+        self, event_id: int, *, with_user_count: bool = None
+    ) -> GuildScheduledEvent:
+        """|coro|
+
+        Returns a :class:`GuildScheduledEvent` with the given ID.
+
+        .. versionadded:: 2.3
+
+        Parameters
+        ----------
+        event_id: :class:`int`
+            The ID to look for.
+        with_user_count: :class:`bool`
+            Whether to include number of users subscribed to each event.
+
+        Raises
+        ------
+        HTTPException
+            The request failed.
+
+        Returns
+        --------
+        :class:`GuildScheduledEvent`
+            The scheduled event.
+        """
+        data = await self._state.http.get_guild_scheduled_event(
+            self.id, event_id, with_user_count=with_user_count
+        )
+        return GuildScheduledEvent(state=self._state, data=data)
+
+    async def create_scheduled_event(
+        self,
+        *,
+        name: str,
+        privacy_level: StagePrivacyLevel,
+        scheduled_start_time: datetime.datetime,
+        entity_type: GuildScheduledEventEntityType,
+        channel_id: Snowflake = MISSING,
+        entity_metadata: GuildScheduledEventMetadata = MISSING,
+        scheduled_end_time: datetime.datetime = MISSING,
+        description: str = MISSING,
+    ) -> GuildScheduledEvent:
+        """|coro|
+
+        Creates a :class:`GuildScheduledEvent`.
+
+        .. versionadded:: 2.3
+
+        Parameters
+        ----------
+        name: :class:`str`
+            The name of the scheduled event.
+        description: :class:`str`
+            The description of the scheduled event.
+        channel_id: :class:`int`
+            The channel id of the scheduled event. Set to ``None`` if changing
+            ``entity_type`` to :class:`GuildScheduledEventEntityType.external`.
```

Comment:
```suggestion
        channel_id: :class:`int`
            The channel id of the scheduled event.
```

Also the annotation is `Snowflake`, docstring says `int`.